lib/src/models/movie.dart

```diff
@@ -18,16 +18,8 @@ class Movie {
   ) {
     var allMovies = parsedJson['results'] as List;
 
-    List<ResultMovie> movieTemp = allMovies
-        .map(
-          (
-            eachMovie,
-          ) =>
-              ResultMovie.fromJson(
-            eachMovie,
-          ),
-        )
-        .toList();
+    List<ResultMovie> movieTemp =
+        allMovies.map((eachMovie) => ResultMovie.fromJson(eachMovie)).toList();
```

Comment:
be careful of this allMovies, it could be null

Reply:
Fixed, now allMovies has a checkpoint that handle if the value were null